dashboard/app/models/user.rb

```diff
@@ -2080,7 +2080,18 @@ def self.track_proficiency(user_id, script_id, level_id)
 
   # The synchronous handler for the track_level_progress helper.
   # @return [UserLevel]
-  def self.track_level_progress(user_id:, level_id:, script_id:, new_result:, submitted:, level_source_id:, pairing_user_ids: nil, is_navigator: false, time_spent: nil)
+  def self.track_level_progress(
```

Comment:
Should we add a test case for the locale in the user model test?

Reply:
We probably should, but we don't really need to. I'll add one.